src/index.ts

```diff
@@ -13,6 +13,8 @@ export {
     type ComponentPath,
     type Meaning,
     type MeaningKeyword,
+    type ExternalResolveAttempt,
+    type ExternalResolveResult,
```

Comment:
Not really a fan of more exported types... might look at how often they actually show up in visible API before release.

Reply:
In fact, we can made this types non-exported, at least `ExternalResolveAttempt` - after update of signature of `addUnknownSymbolResolver` method it is redundant. But I think type `ExternalResolveResult` may be useful for client code. In my use-case, I used this type 4 times in own resolver implementation because of complicated link building logic in our project.

Reply:
> ExternalResolveResult

I agree, this one is useful.